frontend/src/hooks/useValidateLoginForm.ts

```diff
@@ -0,0 +1,57 @@
+import { useVuelidate } from '@vuelidate/core';
+import { email, required, minLength } from '@vuelidate/validators';

```

Comment:
Better to use computed properties for simple validation

Reply:
Done